src/test/java/test/model/KubeResources.java

```diff
@@ -44,6 +47,11 @@ public Deployment getDeployment(final String name) {
                 .getOrElseThrow(() -> new AssertionError("No Deployment found with name " + name));
     }
 
+    public ConfigMap getConfigMap(final String name) {
```

Comment:
Interesting we didn't need it until now :)

Reply:
There are few places where we could rewrite `resources.get(ConfigMap, product.getHelmReleaseName() + "xyz");`. But that is only for bonus points ;)

Reply:
In fact, I failed to find a common way to get a configmap, hence added it to be available for all tests. Yes, we can refactor existing tests afterwards.